buildSrc/src/main/groovy/com/github/okbuilds/core/util/FileUtil.groovy

```diff
@@ -42,11 +42,9 @@ final class FileUtil {
     }
 
     static void copyResourceToProject(String resource, File destination) {
-        try {
-            InputStream inputStream = FileUtil.getResourceAsStream(resource)
-            OutputStream outputStream = new FileOutputStream(destination)
-            outputStream.write(inputStream.bytes)
-            outputStream.close()
-        } catch (Exception ignored) { }
```

Comment:
Copying resource seems to be working now. I have removed the exception catching, easier to catch the issue if it happens again.